workflows/illumina.nf

```diff
@@ -733,7 +699,9 @@ workflow ILLUMINA {
 */
 
 workflow.onComplete {
-    NfcoreTemplate.email(workflow, params, summary_params, projectDir, log, multiqc_report, fail_mapped_reads)
+    if (params.email || params.email_on_fail) {
```

Comment:
Just came up with an idea, could not this be moved to the main pipeline script?

Reply:
If I understand what you mean. I don't think it can because different `workflows` may be calling this function in different ways so it's probably more self-enclosed and tidier to have it in the actual workflow script.